src/frontend/src/forms/StockForms.tsx

```diff
@@ -467,27 +467,31 @@ function StockOperationsRow({
     props.removeFn(props.idx);
   };
 
+  const useChangeFn = (idx: number, key: string, value: any) => {
+    setTimeout(() => props.changeFn(idx, key, value), 0);
+  };
```

Comment:
@wolflu05 do you know of a better way to handle this? calling props.changeFn directly causes a setState in render issue